Updating uninstallation instructions for Vantage (#1600)
DataDog/integrations-extras: vantage/README.md

### Uninstallation

> To remove the Datadog integration from Vantage, visit the [Vantage integrations page][1] and click **remove**. Additionally, you'll want to uninstall this integration from Datadog by clicking the **Uninstall Integration** button below. Once you uninstall this integration, any previous authorizations will be revoked. 

Reviewer: ```suggestion
To remove the Datadog integration from Vantage, navigate to the [Vantage integrations page][1] and click **Remove**. Additionally, uninstall this integration from Datadog by clicking the **Uninstall Integration** button below. Once you uninstall this integration, any previous authorizations are revoked. 
```
